MAINTAINERS

```diff
@@ -138,10 +138,10 @@ W: http://plugins.geany.org/geanyprj.html
 S: Odd Fixes
 
 geanypy
-P: Lex Trotman <elextr@gmail.com>
-M: Lex Trotman <elextr@gmail.com>
```

Comment:
>  IMO it would be best having Matthew Brush at the "+" side of the patch if @codebrainz is willing to maintain it. 

I like having separate repo/issues/build system, etc, which is why I kept GeanyPy as external to Geany-Plugins. I don't think every plugin people write should be forced to be in Geany-Plugins. I don't mind formatting patches for changes to upstream so they can be applied to the fork in Geany-Plugins, but it's going to get harder and harder as the fork has diverged significantly from my repo (ex. the proxy plugin stuff, the geany_functions commit, etc).